examples/svelte/src/App.svelte

```diff
@@ -0,0 +1,66 @@
+<script>
+    import { router } from './router.js';
+    import { uuid } from './utils.js';
+
+    import Header from './Header.svelte';
+    import Footer from './Footer.svelte';
+    import Item from './Item.svelte';
+
+    import "./app.css";
+    import "todomvc-app-css/index.css";
+    import "todomvc-common/base.css";
+
+    let currentFilter = "all";
+    let items = [];
+    let editing = null;
+
+    function addItem(event) {
+        items.push({
+            id: uuid(),
```

Comment:
```suggestion
            id: crypto.randomUUID(),
```

Reply:
We used this function before, but opted out - [here](https://github.com/WebKit/Speedometer/issues/136#issuecomment-1553880792)'s the issue from Speedometer

Reply:
Oh, interesting. What about using `crypto.randomUUID()` for TodoMVC itself and using the `uuid` helper for Speedometer? Or do the implementations need to be identical?